data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1611 @@
+			`	"The Embers saw you safely through." Gestures Torza with a note of appreciation, appearing from a nearby hatch. "The loss of that outpost is unfortunate, but at least most of the Remnant there survived. Thank you for your assistance." He glances at his commlink. Your logistics adjustment has been added to your account, of course, but I suspect that you are interested in pushing forward the progress in studying Ssil Vida. Meet me in the spaceport when you are ready."`
+
+
+
+mission "Remnant: Cognizance 29"
+	name "Transport team to Ssil Vida."
+	description "Torza has tasked you with transporting a team of technicians to Ssil Vida."
+	blocked `You check in with Torza, who glances at his commlink. "You will need a ship with space for 15 people and 30 cubic meters of cargo."`
+	cargo "equipment" 30
+	passengers 15
+	source "Caelian"
+	destination "Ssil Vida"
+	to offer
+		has "Remnant: Cognizance 28: done"
+	on offer
+		conversation
+			`You walk into the cafeteria to find a group of Remnant looking tired but pleased with themselves. Torza is striding among them with a tray filled with bottles, handing them out. "Good work, everyone. And excellent pacing on that run." He turns towards you. "And right on schedule, the captain who will be transporting you over to Ssil Vida." He gestures for you to take a seat and offers you a bottle as well.`
+			choice
+				`	Gesture that you are not thirsty.`
+					goto notthirsty
+				`	Take a bottle and gesture appreciation.`
+			`	You sip from the bottle, and discover that it tastes like some kind of restorative sports drink. You can pick out hints of salt and a few other flavors, but most of it is an indeterminate taste you cannot quite identify. Torza nods in approval as you drink. "Especially in heat like this, hydration is important." He continues weaving around the group handing out the remaining bottles.`
+			choice
+				`	(drink in silence)`
+					goto powerbriefing
+				`	(ask the Remnant next to you what just happened)`
+			`	The Remnant smiles a tiredly. "We did a ten kilometer run in just under thirty seven minutes, wearing our emergency environmental suits and carrying basic survival and maintenance gear. Torza set a hard pace, but we kept up." She takes a sip from her drink. "It is hard work, but endurance is rather important for survival, especially in an unknown environment like Ssil Vida." She takes another sip. "It was an interesting changes of pace. Normally I carry full engineering kit, so this was a 'fast scramble' in comparison. He," she gestures at Torza, "likes to change things up. No two runs are the same when he sets the pace."`
```

Comment:
3.7 minutes per klick? How buff are they?

Reply:
They perform very well.

I used the times from https://runninglevel.com/running-times/10k-times as a basis point, looking at the elite level for 35 year olds. I deliberately picked a time that was exceptional but not unachievable for humans in general. (note the current world-record for that distance and age bracket is 26 min and 30 mins for men and women respectively).

It is intended to be a sort of "Wow, this group of researchers is really fit! Wait a minute, they've talked about having excellent medical care, and there's those symbiotes... Is fitness really what's going on here, or is there something else giving them an edge?"

It should be clear that it *is* an achievement. They are proud of their time, and it was hard work. But it is intentionally suppose to catch the attentive person's attention that it is somewhat incongruous to have an entire technical team who just happens to be elite level runners...